test/format/config.c

```diff
@@ -275,6 +275,13 @@ config_table(TABLE *table, void *arg)
             config_single(table, "btree.value_min=20", false);
     }
 
+    /*
+     * Limit the rows to 1000000 if the realloc exact config is on, otherwise bulk load will take
+     * too much time as we need to do realloc for every new entry appended to an array.
+     */
+    if (GV(DEBUG_REALLOC_EXACT) && TV(RUNS_ROWS) > 1000000)
```

Comment:
I tested 1000000 is fine.

Reply:
```
DEBUG_REALLOC_EXACT is a global value. We might want to check at the global level first because of the way its copied down to each table it can be a touch harder to debug. Take a look at https://github.com/wiredtiger/wiredtiger/pull/8541.
```
We are not changing the value so it is OK. It is a problem if we try to change the config and this will create inconsistency among table configs. But we are not so I think we are safe here.